pom.xml

```diff
@@ -54,7 +54,7 @@
       <dependency>
         <groupId>org.kohsuke</groupId>
         <artifactId>github-api</artifactId>
-        <version>1.309-SNAPSHOT</version>
+        <version>1.313</version>
```

Comment:
You could get rid of that entirely and upgrade Quarkus GitHub App instead as I did in my open PR (but it doesn't contain the workflow upgrades). I released just for you :).